InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -288,7 +292,7 @@ def test_epoch_end(self, outputs: List[Dict[str, Any]]) -> None:  # type: ignore
 
         for key in report_cases.keys():
             print(f"Plotting {key} ...")
-            output_path = Path(fixed_paths.repository_root_directory(), f'outputs/fig/{key}/')
+            output_path = Path(fixed_paths.repository_parent_directory(), f'outputs/fig/{key}/')
```

Comment:
here and at other places where you write into the outputs folder: Good opportunity for DRY (don't repeat yourself). Construct the output folder once, and re-use

Reply:
done